on-premise-docker-registry/README.md

```diff
@@ -0,0 +1,191 @@
+# Overview
+
+Many a time, customers would like to innovate by building applications and deploying them on Kyma runtime. Since Kyma is based on Kubernetes, they must package their application binaries as Docker images. These images then need to be stored on a Docker registry server.
+
+The customers have the option to use a third-party Docker registry hosted on public clouds such as:
+
+- Google Container Registry
+- Azure Container Registry
+- AWS Elastic Container Registry
+- and other vendor offerings
+
+Although this is a reliable option, many a time, customers do not wish to store their artifacts on public clouds. This could be due to various reasons, such as:
```

Comment:
Again "many a time", never heard that before... I would skip it here fully
```suggestion
Although this is a reliable option, customers do not wish to store their artifacts on public clouds. This could be due to various reasons, such as:
```